samples/jwt-server/jwt-server.yaml

```diff
@@ -25,9 +25,106 @@ spec:
   - name: http
     port: 8000
     targetPort: 8000
+  - name: https
+    port: 8443
+    targetPort: 8443
   selector:
     app: jwt-server
 ---
+apiVersion: v1
+kind: ConfigMap
```

Comment:
very nit: store as a Secret.

I know its not "secret" in the sense its in a public github repo, but Secret is literally designed to store TLS certs (there is a specific type for it)

Reply:
good point, I will check that.